src/gui/torrentcontentmodel.cpp

```diff
@@ -113,31 +106,23 @@ namespace
         {
             const QString ext = info.suffix();
             if (!ext.isEmpty()) {
-                const QPixmap pixmap = pixmapForExtension(ext, QSize(32, 32));
-                if (!pixmap.isNull())
-                    return QIcon(pixmap);
+                auto cacheEntry = m_iconCache.find(ext);
+                
+                if (cacheEntry != m_iconCache.end())
+                    return cacheEntry.value();
```

Comment:
personal opinion:
* I would name it `cacheIter` or something like that. `cacheEntry` sounds like its type is `QIcon`.
* `return *cacheIter;`, I'm so used to using `operator*` to dereference an iterator.